index.js

```diff
@@ -61,7 +60,7 @@ exports.decode = function (input, stream) {
     return decoded
   }
 
-  assert.equal(decoded.remainder.length, 0, 'invalid remainder')
+  if (decoded.remainder.length !== 0) throw Error('invalid remainder')
```

Comment:
Is assert not handled correctly by webpack/browserify?

Reply:
Yes.

Reply:
It's a package. Handle for some package is not mission of webpack, browserify or babel.

Reply:
It is kind of weird because this library has been in use for 2 years in browser based projects 😉

Reply:
Can i see some?

Reply:
remix.ethereum.org for one. I suggest to have a look at babeljs to do the browser builds.